### PR TITLE
Resolve #19

### LINKED ISSUE #19: Add missing trivia to the project
* deal with launchSettings.json
* xUnit runner GUID in xproj
* add global.json
* views web.config for IIS

### PR DESCRIPTION
**Pull request by AlekseyMartynov**

### CODE REVIEW

File: net/DevExtreme.AspNet.Data.Tests/DevExtreme.AspNet.Data.Tests.xproj [hunk 1]
```diff
@@ -15,5 +15,8 @@
   <PropertyGroup>
     <SchemaVersion>2.0</SchemaVersion>
   </PropertyGroup>
+  <ItemGroup>
+    <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
+  </ItemGroup>
```
Comment (IlyaKhD) [inline net/DevExtreme.AspNet.Data.Tests/DevExtreme.AspNet.Data.Tests.xproj]: This is what VisualStudio automatically adds to tests projects config?

**Comment by IlyaKhD** [comment]
:shipit:

File: net/DevExtreme.AspNet.Data.Tests/DevExtreme.AspNet.Data.Tests.xproj (hunk 1 above)
Comment (AlekseyMartynov) [inline net/DevExtreme.AspNet.Data.Tests/DevExtreme.AspNet.Data.Tests.xproj]: Yes, it is

**Pull request merged**